gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/rules/AccumulativeExecutionEngineRule.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Yes I understand the intention after looking at the code but the use of 'Accumulative' is not appropriate for this purpose in my opinion. It is more like a visitor than an accumulator ( compare Spark's accumulators ). May be it is fine. Leave it for now but think about it :-)